app/src/main/java/com/github/sdp_begreen/begreen/data/FeedMediator.kt

```diff
@@ -0,0 +1,5 @@
+package com.github.sdp_begreen.begreen.data
+
+class FeedMediator {
```

Comment:
Mhhhh

Reply:
Yeah it's pretty empty for now =D. I have good hopes it will be filled with useful stuff when I'll finally have tamed paging 3 with my final PR